src/Illuminate/View/Compilers/ComponentTagCompiler.php

```diff
@@ -776,13 +776,47 @@ protected function attributesToString(array $attributes, $escapeBound = true)
     {
         return collect($attributes)
                 ->map(function (string $value, string $attribute) use ($escapeBound) {
-                    return $escapeBound && isset($this->boundAttributes[$attribute]) && $value !== 'true' && ! is_numeric($value)
-                                ? "'{$attribute}' => \Illuminate\View\Compilers\BladeCompiler::sanitizeComponentAttribute({$value})"
-                                : "'{$attribute}' => {$value}";
+                    if (
+                        (! $escapeBound) ||
+                        (! isset($this->boundAttributes[$attribute])) ||
+                        ($value === 'true') ||
+                        is_numeric($value)
+                    ) {
+                        return "'{$attribute}' => {$value}";
+                    }
+
+                    return "'{$attribute}' => \Illuminate\View\Compilers\BladeCompiler::sanitizeComponentAttribute({$value})";
```

Comment:
And you can pretty much ignore this set of changes, it's just a readability refactor that I can revert if you want

Reply:
Yes revert readability stuff if you don't mind. Makes it a bit easier for me.

Reply:
Done